packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -123,11 +124,52 @@ class CupertinoCheckbox extends StatefulWidget {
 
   /// The color to use when this checkbox is checked.
   ///
+  /// If [fillColor] returns a non-null color in the [WidgetState.selected]
+  /// state, it will be used instead of this color.
+  ///
   /// Defaults to [CupertinoColors.activeBlue].
   final Color? activeColor;
 
+  /// {@template flutter.material.checkbox.fillColor}
```

Comment:
`@template` is used to create documentation macros. So, if creating a macro is the desired outcome, then it should have the name `flutter.cupertino.CupertinoCheckbox.fillColor`.